web/docs/operators/set.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_custom_props:
+  operator:
+    transformation: true
+---
+
+# set
+
+Upserts fields in events.
+
+## Synopsis
+
+```
+set <field=operand>...
+```
+
+## Description
+
+The `set` operator appends a specified list of fields to the input. If any of
+the fields already exist in the input event, they are replaced again.
```

Comment:
```suggestion
The `set` operator sets a list of fields to the given values. It overwrites old
values of fields matching the `field` expression, or creates new fields of a
given name otherwise.
```